.github/workflows/sync-template.yml

```diff
@@ -0,0 +1,76 @@
+name: Sync branch to template
+
+on:
+  workflow_dispatch:
+    inputs:
+      additional_files:
+        description: 'Comma-separated list of additional files to sync (i.e ".github/workflows/build.yml tsconfig.json")'
+        required: false
+  schedule:
+    - cron: '14 0 1 * *'
+
+jobs:
+  sync:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      pull-requests: write
+    steps:
+      - name: Check if repository is ts-template
+        run: |
+          if [[ "${{ github.repository }}" == "ubiquity/ts-template" ]]; then
+            echo "Skipping sync: this is the template repository."
+            exit 0
+          fi
+
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Get GitHub App token
+        uses: tibdex/github-app-token@v1.7.0
+        id: get_installation_token
+        with:
+          app_id: ${{ secrets.APP_ID }}
+          private_key: ${{ secrets.APP_PRIVATE_KEY }}
+
+      - name: Sync branch to template
+        env:
+          GH_TOKEN: ${{ steps.get_installation_token.outputs.token }}
+          WHITELIST_FILES: ".github/workflows/build.yml tsconfig.json"
+          ADDITIONAL_FILES: ${{ github.event.inputs.additional_files }}
```

Comment:
Is this a list? How exactly is this defined? What's the value in your qa

Reply:
For example in [this QA](https://github.com/zugdev/test-ts-template-sync/actions/runs/11768762168/job/32778949745), I dispatched with the following inputs: .eslintrc,.prettierrc,.github/

![image](https://github.com/user-attachments/assets/5505f3c7-c6fa-4ced-943c-819574a004c8)

With the intention of synchronizing .eslintrc, .prettierrc and the entire .github/ directory, you can see it works as intended: 

![image](https://github.com/user-attachments/assets/9bbaed6a-23c3-4e2f-8eec-7be1b2a05efb)

Reply:
I'll improve the description, it's a little bit confusing